.github/workflows/macos-build.yaml

```diff
@@ -4,15 +4,18 @@ on:
   pull_request:
   push:
     branches:
-      - master
+      - macos-11-runner
```

Comment:
Don't forget to change this back to master.
```suggestion
      - master
```

Reply:
@nwt: Thanks for the reminder!